cms-docker-gh-issues.py

```diff
@@ -69,7 +69,8 @@
 )
 
 if args.comment == False:
-    pulls_curl = "curl -s 'https://api.github.com/repos/%s/pulls?q=+is:open+label:%s'" % (
+
+    pulls_curl = "curl -s 'https://api.github.com/repos/%s/issues?state=open&labels=%s'" % (
```

Comment:
by the way, why are you not using `https://api.github.com/repos/%s/issues` API call  for `issues_curl` ?

Reply:
For issues I was using the same call as in https://github.com/cms-sw/cms-bot/blob/master/create-gh-issue.py#L63, but I can modify it :)